compiler/src/iree/compiler/GlobalOptimization/DemoteContractionInputsToBF16.cpp

```diff
@@ -21,12 +21,24 @@ namespace mlir::iree_compiler::GlobalOptimization {
 
 namespace {
 
+const std::string kDemoteAll = "all";
+const std::string kDemoteOnlyConv = "conv";
+const std::string kDemoteOnlyMatmul = "matmul";
+const std::string kDemoteNone = "none";
```

Comment:
I think we can use enum to manage it better; we can get rid of magic strings. Can you add the `enum class` to Passes.h, and use `clEnumValN` in td file? Here is an example, see `Passes.h` and `Passes.td`: https://github.com/iree-org/iree/commit/e6d8aa70904f182881f388137708c07e5127483e

Reply:
Done, please review.